Bio/Align/__init__.py

```diff
@@ -1229,6 +1232,22 @@ def _format_psl(self):
         line = "\t".join(words) + "\n"
         return line
 
+    def format(self):
+        """Create a human-readable representation of the alignment (DEPRECATED).
+
+        This method is deprecated; instead of alignment.format(), please use
+        format(alignment).
+        """
+        warnings.warn("""\
```

Comment:
Same comment about triple quotes applies here too.

Reply:
Yes let's handle this later. I'd like to commit this before master changes again.